### DISCUSSION TITLE
Issue with 'Generating Content' tutorial

### DISCUSSION THREAD

**Discussion by virenb** [blitz-js/blitz#723] [General] [answered]
Hello,

In the 'Generating Content', I was running into an error when trying to generate the Choice model.

`blitz generate resource choice text votes:int:default[0] belongsTo:question`

I was getting an error in my terminal, `zsh: no matches found: votes:int:default[0]`

What worked for me was removing the `[0]` from `votes:int:default[0]` then I manually added it back in the schema.prisma file and was able to follow the rest of the tutorial.

FWIW, this is my first time trying out Blitz and I haven't worked with SQLite in the past much. I'd be glad to provide any further information if needed.

**Comment by flybayer** [comment, marked as answer]
Hey @virenb! Unfortunately that is expected behavior with zsh because zsh doesn't like the brackets. The solution is to wrap it in quotes like `"votes:int:default[0]"`.

That's definitely sub-optimal and we should try to find a way to do this that works on all terminals.

I created an issue for this: https://github.com/blitz-js/blitz/issues/724

**Reply by virenb** [reply]
Thanks a lot for your message and action, learned something new about zsh today.
